packages/libs/src/sdk/api/grants/GrantsApi.ts

```diff
@@ -39,6 +48,43 @@ export class GrantsApi {
     })
   }
 
+  /**
+   * When user authorizes another user to perform actions on their behalf.
+   * The grant has to be approved by the proposed manager.
+   */
+  async addManager(params: AddManagerRequest) {
+    const { userId, managerUserId } = await parseParams(
+      'addManager',
+      AddManagerSchema
+    )(params)
+    let managerUser: User | undefined
+    try {
+      managerUser = (
+        await this.usersApi.getUser({
+          id: encodeHashId(managerUserId)!
+        })
+      ).data
+      if (!managerUser) {
+        throw new Error()
+      }
+    } catch (e) {
+      throw new Error(
+        '`managerUserId` passed to `addManager` method is invalid.'
+      )
+    }
+
+    return await this.entityManager.manageEntity({
+      userId,
+      entityType: EntityType.GRANT,
+      entityId: 0, // Contract requires uint, but we don't actually need this field for this action. Just use 0.
+      action: Action.CREATE,
+      metadata: JSON.stringify({
+        grantee_address: managerUser!.ercWallet
```

Comment:
Should we just check this and throw? It's going to fail on the indexing side if the wallet isn't valid.

Reply:
Sorry what should we check? The wallet validity? Since the value is coming from the API it should be valid.

Reply:
If we're using `mangerUser!`, that would indicate that the typing says the user could be undefined/null but we're ignoring that and forcing it. I'd advocate that we check it above and throw an exception if it's actually undefined and then proceed here without the exclamation point.